populateDB/data/addLegacyCrossings.js

```diff
@@ -9,128 +9,26 @@ const LegacyCrossingsCsvPath = `${__dirname}/legacyCrossings.csv`;
-}
-
-async function addCrossingToCommunity(lokka, crossingId, communityId) {
-  const response = await lokka.send(
-    `
-    mutation($crossingId:Int!, $communityId:Int!) {
-      addCrossingToCommunity(input: {crossingId:$crossingId, communityId:$communityId}) {
-        crossing {
-          id
-          legacyId
-        }
-      }
-    }
-    `,
-    {
-      crossingId: crossingId,
-      communityId: communityId,
-    },
-  );
-
-  return response.addCrossingToCommunity.crossing;
-}
-
-async function getExistingCrossings(lokka) {
-  const response = await lokka.send(
-    `
-    {
-      allCrossings {
-        nodes {
-          id
-        }
-      }
-    }
-  `,
-  );
-
-  return response.allCrossings.nodes;
-}
-
-async function removeCrossing(lokka, crossingId) {
-  const response = await lokka.send(
-    `
-    mutation($crossingId:Int!) {
-      removeCrossing(input:{crossingId:$crossingId}) {
-        crossing {
-          id
-        }
-      }
-    }
-  `,
-    {
-      crossingId: crossingId,
-    },
-  );
-
-  return response.removeCrossing.crossing.id;
-}
-
-async function processCrossings() {
-
-  const lokka = await getAuthorizedLokka('superadmin@flo.ods', 'texasfloods');
-
-  let crossings = await loadCsv(LegacyCrossingsCsvPath);
-
-  console.log('Removing Existing Crossings');
-  const existingCrossings = await getExistingCrossings(lokka);
-
-  for (crossing of existingCrossings) {
-    const removedCrossingId = await removeCrossing(lokka, crossing.id);
-    // console.log(`Removed existing crossing with id: ${removedCrossingId}`);
-  }
-
-  console.log('Adding Legacy Crossings');
-  let legacyToNewMap = {};
+async function addLegacyCrossings(client) {
+  const crossings = await loadCsv(LegacyCrossingsCsvPath);
+  let header = true;
+  // Hack to get around graphql infrastructure
+  await client.query(`select set_config('jwt.claims.role','floods_super_admin', false)`);
+  await client.query(`select set_config('jwt.claims.community_id','floods_community_admin', false)`);
+  await client.query(`select set_config('jwt.claims.user_id','1', false)`);
+  const queryText = `select floods.seed_legacy_crossing($3::text, $4::text, $1::integer, $6::decimal, $5::decimal, $7::text, $2::integer, $8::integer)`
   for (crossing of crossings) {
-    const newId = legacyToNewMap[crossing.legacyId];
-
-    if (!newId) {
-      const newCrossing = await addCrossing(lokka, crossing);
-      legacyToNewMap[newCrossing.legacyId] = newCrossing.id;
-      // console.log(`Added legacy crossing ${newCrossing.legacyId} to DB`);
-    } else {
-      const updatedCrossing = await addCrossingToCommunity(
-        lokka,
-        newId,
-        crossing.communityId,
-      );
-      // console.log(
-      //   `Added legacy crossing ${updatedCrossing.legacyId} to community ${
-      //     crossing.communityId
-      //   }`,
-      // );
+    if (header) {
+      header = false;
+      continue
     }
+    crossing[1] = crossing[1] || null;
```

Comment:
might be nice to add a little comment in here describing what each of these does. I'm assuming we're dealing with null-***ish*** vs actual null here but clarity can't hurt